org.gnome.Yelp.desktop.in.in

```diff
@@ -1,10 +1,10 @@
 [Desktop Entry]
-_Name=Help
-_Comment=Get help with GNOME
-_Keywords=documentation;information;manual;help;
+Name=Help Center
```

Comment:
```suggestion
Name=Help
```

The name change is done by eos-shell-content, at least in principle. https://github.com/endlessm/eos-shell-content/blob/d04b738c8ca6a1f92466439b054184fbab113446/bundle/desktops/org.gnome.Yelp.desktop#L3

Reply:
Addressed.